dbus.c

```diff
@@ -169,6 +175,7 @@ static zend_object_handlers dbus_object_handlers_dbus_struct, dbus_object_handle
 typedef struct _php_dbus_obj {
 	DBusConnection *con;
 	int             useIntrospection;
+	int             default_timeout_ms;
```

Comment:
Would you mind using the same capitalisation as the already existing parameter (`useIntrospection`).

Reply:
SI units and their prefixes are case-sensitive. ms is milliseconds, `MS` would be mega-siemens. However I do see the value of consistency with the other struct members. I'll add a comment that makes it clear what the actual unit is. Could also spell it out (`defaultTimeoutMilliseconds`) if you prefer.

Reply:
> Could also spell it out (defaultTimeoutMilliseconds) if you prefer.

I don't really mind it looking like mega-siemens (As `MS`), but spelling it out is fine too.

> Would it make sense to rename useIntrospection to use_introspection instead?

Yes, that would make sense :-). Can you do that as a separate commit? I think the original capitalisation came from the DBUS spec at some point.